roles/bootstrap-os/tasks/bootstrap-oraclelinux.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Download Oracle Linux public yum repo
+  get_url:
+    url: "{{ repo_url }}"
+    dest: "{{ repo_dest }}"
+  vars:
+    repo_dest: /etc/yum.repos.d/public-yum-ol7.repo
+    repo_url: https://yum.oracle.com/public-yum-ol7.repo
+
+- name: Enable Oracle Linux repo
+  ini_file:
+    dest: /etc/yum.repos.d/public-yum-ol7.repo
+    section: "{{ item }}"
+    option: enabled
+    value: 1
+  with_items:
+    - ol7_latest
+    - ol7_addons
+    - ol7_developer_EPEL
+
+- name: Install packages requirements for bootstrap
+  yum:
+    name:
+      - container-selinux
+      - python-pip
```

Comment:
What do you need pip for?

Reply:
Not entirely sure why I put python-pip here. it was almost half a year ago:). I have just tested it without  python-pip and it worked fine. I will remove it.

Reply:
@MarkusTeufelberger , please check again.